vispy/scene/widgets/widget.py

```diff
@@ -5,6 +5,7 @@
 from __future__ import division
 
 import numpy as np
+from vispy.visuals import _BorderVisual
```

Comment:
If `_BorderVisual` is meant to be private, then there is no need for it to be imported to the `vispy.visuals` namespace. However if there's any possibility that it will be useful elsewhere, then it should not be private.